src/python/grpcio_reflection/grpc_reflection/v1alpha/reflection.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016, Google Inc.
```

Comment:
Project policy is that we don't update copyright year when modifying existing files (but new files, including branches of old files, get the current year).

I apologize that this isn't written down anywhere, I don't think. :-(